tools/releaseBuild/images/CentOS/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/powershell:centos-7
+FROM mcr.microsoft.com/powershell:centos-7
```

Comment:
Is it possible to verify the validity of this URL through a web UI? I don't have Docker setup anywhere, and going to mcr.microsoft.com redirects to a GitHub repository.

Reply:
It would be a complex series of rest calls to verify:
![image](https://user-images.githubusercontent.com/10873629/118164004-79ef5280-b3d7-11eb-9d4f-17eed191e2a5.png)

Reply:
Perfect!